lib/vagrant-service-manager.rb

```diff
@@ -8,7 +8,7 @@
 require 'vagrant-service-manager/command'
 require 'vagrant-service-manager/os'
 
-module Vagrant
+module VagrantPlugins
```

Comment:
This is for example a perfect example where multiple commits would make sense to help reviewing the overall changes and separating a valid refactoring from the actual changes required for the debug logging.

My guess is that the change from _Vagrant_ to _VagrantPlugins_ is unrelated and works standalone.